testpilot/frontend/views.py

```diff
@@ -1,6 +1,12 @@
+from django.conf import settings
 from django.shortcuts import render
+from django.views import static
 
 
 def index(request, url=''):
     return render(request, 'testpilot/frontend/index.html',
                   {'user_id': request.user.email if not request.user.is_anonymous() else ''})
+
+
+def contribute_json(request):
```

Comment:
Same thing, here - [`base/views.py`](https://github.com/mozilla/testpilot/blob/master/testpilot/base/views.py) might be a better home